Improve readme (#104)
zeit/swr: README.md

}
```

> _If you want to pass varibales to a GraphQL query, check out [Multiple Arguments](#multiple-arguments)._

Reviewer: ```suggestion
_If you want to pass variables to a GraphQL query, check out [Multiple Arguments](#multiple-arguments)._
```

Author: Thank you!

}
```

### Multiple Arguments

> In some scenarios, we need to pass multiple arguments (can be any value or object) to the `fetcher` function. For example:

Reviewer 2: ```suggestion
In some scenarios, it's useful pass multiple arguments (can be any value or object) to the `fetcher` function. For example:
```

```js
const token = props.token

useSWR('/api/data', url => fetchWithToken(url, token))
```

**This is incorrect**. Because the identifier of the data is `'/api/data'`, which is also the index of the cache. 
When `token` changes, SWR will still treat it as the same key and request. 

Instead, you can use an array as the `key` parameter, which contains multiple arguments of `fetcher`:

```js
useSWR(['/api/data', token], fetchWithToken)
```

> It solves the problem. The identifier of this request is now the combination of both values. SWR **shallowly** compares

Reviewer 2: ```suggestion
This solves the problem. The identifier of the request is now the combination of both values. SWR **shallowly** compares
```
